mithril.js

```diff
@@ -49,9 +49,12 @@ var m = (function app(window, undefined) {
 	 *
 	 */
 	function m(tag, pairs) {
-		for (var args = [], i = 1; i < arguments.length; i++) {
-			args[i - 1] = arguments[i];
+		// JSX compatibility with 'null' {pairs} argument
+		var i = pairs === null ? 2 : 1;
+		for (var args = []; i < arguments.length; ++i) {
+			args.push(arguments[i]);
 		}
+
 		if (isObject(tag)) return parameterize(tag, args);
```

Comment:
I think we should review all cases:
``` js
/// Base Mithril cases:
// arguments.length === 1, pairs is undefined
m('div')
// arguments.length === 2+, pairs is Object, next arguments is undefined or Array (the single third argument) or one or more functions
m('div', {id: '1'}, ...)
// arguments.length === 2, pairs is Array
m('div', [m('div')])
// arguments.length === 2+, pairs is function, as next arguments should too
m('div', m('div'))
// arguments.length === 2+, pairs is String
m('div', 'Hello')
```

JSX emitter always emits function call with 2 or 3 arguments. In case of absent `pairs`, it emits `null` instead.
So for JSX, cases are following:
``` js
m('div', {})
m('div', null, m('div'))
m('div', {}, m('div'))
```

So, if second argument is `null`, Mithril just could consider that second argument is not second, but third one.
I'll fix the PR.